Assignment2/Cristinalonso29-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    @prefix : <http://example.org/elements>,
+    "xsd": <http://www.w3.org/2001/XMLSchema#>.

```

Comment:
They shouldn't be defined between <>
Also, the @prefix isn't a tag of JSON-LD, thus define it well and without @ . Finally, there is something missing at the end of the URI of "prefix"
Remove the point at the end of the "xsd" definition.
Please, before resubmiting valitate your file.